query-languages/m/m-spec-types.md

```diff
@@ -241,6 +241,13 @@ type function (x as text) as number
 type function (y as number, optional z as text) as any
 ```
 
+Specifying a parameter as optional also makes its type nullable. The following create identical function types:
```

Comment:
It seems like this should go at the end of the section, rather than here.

Reply:
Thanks, @ehrenMSFT! Moved. How does it look now?